src/utils/constants.js

```diff
@@ -1,2 +1,10 @@
 // eslint-disable-next-line import/prefer-default-export
 export const NO_DATA = 0;
+export const MASINDI_SECONDARY_ENROLLMENT_URL =
+  'https://raw.githubusercontent.com/devinit/ug-district-dashboard-viz/dev/public/assets/data/masindi/secondary_school_enrolment.csv';
+export const MASINDI_PRIMARY_ENROLLMENT_URL =
+  'https://raw.githubusercontent.com/devinit/ug-district-dashboard-viz/dev/public/assets/data/masindi/primary_school_enrolment.csv';
+export const KAYUNGA_SECONDARY_ENROLLMENT_URL =
+  'https://raw.githubusercontent.com/devinit/ug-district-dashboard-viz/dev/public/assets/data/kayunga/learners_enrolment_secondary.csv';
+export const KAYUNGA_PRIMARY_ENROLLMENT_URL =
+  'https://raw.githubusercontent.com/devinit/ug-district-dashboard-viz/dev/public/assets/data/kayunga/primary_and_kindergaten_enrolment.csv';
```

Comment:
Why are we using this here? The idea is to have everything configured via the CMS. Do let me know if you need help to understand how this works in case you haven't figured it out yet. It was in the docs (pdf) from Thatcher that I shared with you. You can also look at the index.html file in this repo for live examples.

Reply:
Moved this to the application state under topics. Created a health topic with number of health facilities indicator.